design/architecture/api.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "API layer"
+API layer
```

Comment:
```suggestion
title: "API layer"
```

This first top part is called the YAML header, and contains the metadata for the document when it is built to the website. So we need both the `---` separators as well as the `key: value` pairing, in this case `title: `.